hack/generate/csv.sh

```diff
@@ -14,7 +14,7 @@ registry_host='registry.ci.openshift.org'
 registry="${registry_host}/openshift"
 client_version="$(metadata.get dependencies.cli)"
 kn_event="${registry_host}/knative/release-${client_version%.*}:client-plugin-event"
-rbac_proxy="registry.ci.openshift.org/origin/4.7:kube-rbac-proxy"
+rbac_proxy="registry.ci.openshift.org/origin/4.12:kube-rbac-proxy"
```

Comment:
that is the same as `quay.io/brancz/kube-rbac-proxy:v0.15.0` ?

asking b/c of this mapping here: https://github.com/openshift/release/blob/master/core-services/image-mirroring/openshift/mapping_origin_4_12#L174C55-L174C100   :sweat_smile:

Reply:
It has the flag. That's a good news. And I hope it's from OpenShift midstream. 

In a product release that's replaced by `registry.redhat.io` variant. 

https://github.com/openshift/kube-rbac-proxy/tree/release-4.12

Reply:
Hm, do we maybe (in a followup-PR) want to link the version to https://github.com/openshift-knative/serverless-operator/blob/main/olm-catalog/serverless-operator/project.yaml#L31 instead of having a version in that bash file? @mgencur

Reply:
I've seen on 1.31 it's already derived from ocp min version metadata.

Reply:
https://github.com/openshift-knative/serverless-operator/blob/main/hack/generate/csv.sh#L17